python/cudf/cudf/core/column/struct.py

```diff
@@ -190,3 +192,35 @@ def field(self, key):
             return self._return_or_inplace(self._column.children[pos])
         else:
             return self._return_or_inplace(self._column.children[key])
+
+    def explode(self):
+        """
+        Return a DataFrame whose columns are the fields of this struct Series.
+        Note that a copy of the columns is made.
+
+        Examples
+        --------
+        >>> s
+        0    {'a': 1, 'b': 'x'}
+        1    {'a': 2, 'b': 'y'}
+        2    {'a': 3, 'b': 'z'}
+        3    {'a': 4, 'b': 'a'}
+        dtype: struct
+
+        >>> s.struct.explode()
+           a  b
+        0  1  x
+        1  2  y
+        2  3  z
+        3  4  a
+        """
+        return cudf.DataFrame._from_data(
+            cudf.core.column_accessor.ColumnAccessor(
+                {
+                    name: col.copy()
```

Comment:
```suggestion
                    name: col.copy(deep=True)
```
I always forget the default behavior of this function, and copies are important enough to us that I would prefer to make it explicit in any case. I've actually wondered if we should get rid of the default parameter altogether since from what I've seen it's not clear that there is a clearly preferred alternative (although I may also be mixing this up with the `Frame` class's corresponding method). Also out of scope for this PR, just a thought.

Reply:
I added `deep=True` -- ditto below.